src/java.desktop/share/classes/javax/swing/JFileChooser.java

```diff
@@ -1412,7 +1412,8 @@ public boolean isMultiSelectionEnabled() {
 
     /**
      * Returns true if hidden files are not shown in the file chooser;
-     * otherwise, returns false.
+     * otherwise, returns false. The default value of this property is derived from native platform,
+     * unless set explicitly by the developer.
```

Comment:
Looks like the rest of the file uses the terms "by the programmer" and "This default depends on the operating system.",
also please split the long line up to 80 chars.

Reply:
Sure @mrserb , updated the PR.

Reply:
Not sure the "native" is needed in the new version, any other thoughts from anyone?

Reply:
Yeah, "native OS" and "OS" gives same meaning I guess..